src/main/java/dk/bec/polonez/reservationsystem/service/AuthService.java

```diff
@@ -0,0 +1,33 @@
+package dk.bec.polonez.reservationsystem.service;
+
+import dk.bec.polonez.reservationsystem.model.User;
+import dk.bec.polonez.reservationsystem.repository.UserRepository;
+import org.springframework.http.HttpStatus;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.stereotype.Service;
+import org.springframework.web.server.ResponseStatusException;
+
+@Service
+public class AuthService {
+
+    private final UserRepository userRepository;
+
+    public AuthService(UserRepository userRepository) {
+        this.userRepository = userRepository;
+    }
+
+    public boolean isAdminLoggedIn() {
+        return getCurrentUser().getRole().getName().equals("SYS_ADMIN");
+    }
+
+    public boolean isPoLoggedIn() {
```

Comment:
**advice** "Po" is an abbreviation and can be less readable/less intuitive. What about using full name: ```isPlaceOwnerLoggedIn```